frontend/config/routes.rb

```diff
@@ -1,9 +1,5 @@
 Spree::Core::Engine.add_routes do
 
-  root :to => 'home#index'
```

Comment:
@priyank-gupta - those routes are in core right now - we need to access them for proper email generation (we're linking to home and products in emails) - better UX